draft-ietf-quic-transport.md

```diff
@@ -2123,7 +2123,21 @@ unable to due to connection-level flow control (see {{blocking}}). BLOCKED
 frames can be used as input to tuning of flow control algorithms (see
 {{fc-credit}}).
 
-The BLOCKED frame does not contain a payload.
+The BLOCKED frame is as follows:
+
+~~~
+ 0                   1                   2                   3
+ 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                         Offset (i)                         ...
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+
```

Comment:
You need another `~~~` here to end the figure.

Reply:
Fixed.